development/nautobot_config.py

```diff
@@ -61,13 +61,20 @@
 
 TESTING = len(sys.argv) > 1 and sys.argv[1] == "test"
 
-# Django Debug Toolbar
-DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda _request: DEBUG and not TESTING}
-
-if DEBUG and "debug_toolbar" not in INSTALLED_APPS:  # noqa: F405
-    INSTALLED_APPS.append("debug_toolbar")  # noqa: F405
-if DEBUG and "debug_toolbar.middleware.DebugToolbarMiddleware" not in MIDDLEWARE:  # noqa: F405
-    MIDDLEWARE.insert(0, "debug_toolbar.middleware.DebugToolbarMiddleware")  # noqa: F405
+if DEBUG:
+    try:
+        # pylint: disable-next=import-outside-toplevel
+        import better_exceptions
```

Comment:
Should I keep `better-exceptions` here? It helps with debugging, displays arguments passed within stack trace output.

Reply:
if it comes from our best developer practices, let's do it :)

Reply:
Will remove it here and open an issue to Nautobot Core

Reply:
https://github.com/nautobot/nautobot/issues/4296